app/src/editor/shared/components/modules/ExportCSV.jsx

```diff
@@ -46,6 +48,20 @@ export default class ExportCSVModule extends React.Component<Props, State> {
         }
     }, 250)
 
+    downloadFile = async (url: string, filename: string) => {
+        const result = await api().get(url, {
```

Comment:
We should maybe use the `formatApiUrl` helper here to wrap the url if the base path changes at some point.